### PR TITLE
Feature/remove libphonenumber cx 4382

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
In order to validate the SMS country code `libphonenumber-js` has been imported in the main bundle, resulting in a 20% size increase. This library should only be used in chunks imported on-demand, such as the crossDevice bundle.

# Solution
Only import the module needed from `react-phone-number-input` library. I also reduced the bundle size limit check to 150kb, so in case this library is moved into the main bundle by mistake, the build will fail.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-872-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-872-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-872-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-872-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://872-pr-onfido-sdk-ui-onfido.surge.sh

File: src/index.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
 import { h, render, Component } from 'preact'
 import { Provider as ReduxProvider } from 'react-redux'
 import EventEmitter from 'eventemitter2'
-import { isSupportedCountry } from 'libphonenumber-js'
+import {getCountryCodes} from 'react-phone-number-input/modules/countries'
```
Comment (kopijunkie) [inline src/index.js]: Very Minor: missing spaces between curly braces for consistency

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-872-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-872-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://872-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-872-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-872-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://872-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**